draft-sharabayko-mops-srt.md

```diff
@@ -2057,127 +2057,586 @@ i.e., carried by an incoming ACK.
-Every 10 ms the sender receives the latest values of RTT and RTT variance,
-Available Buffer Size, Packets Receiving Rate and Estimated Link Capacity.
-Upon reception of the NAK packet ({{ctrl-pkt-nak}}) the sender can detect
-packet losses during the transmission.
-These mechanisms provide a solid background for various congestion control
-algorithms.
-
-Given that SRT can operate in live and file transfer modes, there are two possible groups
-of congestion control algorithms.
-
-### Live Congestion Control (LiveCC)
-
-For live transmission mode ({{transmission-mode-live}}) the congestion control algorithm
-does not need to control the sending pace of the data packets, as the sending timing
-is provided by the live input source. It is mainly a reactor on network events. Certain
-limitations on the minimal inter-sending time of consecutive packets can be applied in order
+# SRT Packet Pacing and Congestion Control
+
+SRT provides certain mechanisms for the sender to get a feedback
+from the receiving side through acknowledgement (ACK) packets
+({{ctrl-pkt-ack}}). Every 10 milliseconds the sender receives
+the latest values of RTT, RTT variance, available buffer size,
+receiving rate and estimated link capacity. Upon reception of
+an NAK packet ({{ctrl-pkt-nak}}) the sender detects packet loss
+during the transmission. These mechanisms provide a solid background
+for various congestion control algorithms integration in the
+SRT protocol.
+
+Given that SRT can operate in live and file transfer modes, there are
+two groups of congestion control algorithms defined in SRT: live
+congestion control (LiveCC) and file transfer congestion control
+(FileCC).
+
+## SRT Packet Pacing and Live Congestion Control (LiveCC) {#liveCC}
+
+To ensure smooth video playback of a receiving peer, SRT in live mode
+({{transmission-mode-live}}) must control the sender's buffer level to
+prevent overfill and depletion. Output pace control (OPC) module is
+designed to send packets as fast as they are submitted by the
+application while maintaining relatively stable buffer level.
+While this looks like a simple problem, the sender's constituent
+of the Automatic Repeat Request (ARQ) system between
+input and output of the SRT sender adds some complexity.
+
+SRT needs a certain amount of bandwidth overhead in order to have
+space to insert retransmitted packets without affecting the SRT
+sender's main stream output rate too much so that packets can be
+transmitted normally.
+
+In live mode, the timing of sending original packets is defined by
+the live input (e.g., encoder or streaming application).
+The goal of OPC module is to recreate the sender's input on the decoder
+(or receiving application) output, such that the act of retransmitting
+packets is transparent, as if they never had to be resent in
+the first place.
+
+It is done by adjusting the maximum allowed bandwidth MAX_BW
+({{maxbw}}) which limits the bandwidth usage by SRT and is used
```

Comment:
I think my first version is good - it limits the bandwidth usage of SRT by means of introducing maximum allowed bandwidth MAX_BW. Otherwise, "maximum allowed" is written twice in one sentence and it complicates it. I will keep the original version.